packages/wdio-shared-store-service/tests/launcher.test.ts

```diff
@@ -1,4 +1,4 @@
-import { writeFile, deleteFile } from '../src/utils'
+import type { SharedStoreServiceCapabilities } from '../build/types'
```

Comment:
better to directly pull the type from the TypeScript file
```suggestion
import type { SharedStoreServiceCapabilities } from '../src/types'
```